pkg/csi/service/node.go

```diff
@@ -363,20 +467,13 @@ func (s *service) NodeGetInfo(
 	}
 	cfg, err := cnsconfig.GetCnsconfig(cfgPath)
 	if err != nil {
-		if os.IsNotExist(err) {
```

Comment:
Why are we removing this code block?
This is added in this MR- https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/96/files to fix https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/95

Reply:
Seems something went wrong with my repo, I don't remember having deleted these lines...

Reply:
Added missing lines